cms/envs/aws.py

```diff
@@ -255,6 +255,15 @@
 MODULESTORE = convert_module_store_setting_if_needed(AUTH_TOKENS.get('MODULESTORE', MODULESTORE))
 CONTENTSTORE = AUTH_TOKENS['CONTENTSTORE']
 DOC_STORE_CONFIG = AUTH_TOKENS['DOC_STORE_CONFIG']
+
+try:
+    from pymongo import ReadPreference
+except ImportError:
+    pass
+else:
+    if 'read_preference' in DOC_STORE_CONFIG:
+        DOC_STORE_CONFIG = getattr(ReadPreference, DOC_STORE_CONFIG['read_preference'])
+
```

Comment:
I think we are going to move to 'NEAREST' so that we can spread the read load across the two instances.